swift_browser_ui_frontend/src/components/BrowserSecondaryNavbar.vue

```diff
@@ -47,9 +47,10 @@
           <c-icon-button text>
             <i class="mdi mdi-information-outline" />
           </c-icon-button>
-          <span class="tooltip-content">
-            {{ $t("message.share.share_id_tooltip") }}
-          </span>
+          <span 
+            class="tooltip-content" 
+            v-html="$t('message.share.share_id_tooltip')"
```

Comment:
this should be fixed

Reply:
we had a similar discussion here: https://github.com/CSCfi/swift-browser-ui/pull/506/files#diff-8a59a01ae17b67fe35e4691bcc15ec074521bfbde0f21ff8c1a2863bb3b19120

Reply:
i will have the same suggestion as last time, sanitizing that input, this https://github.com/cure53/DOMPurify seems useful
`var clean = DOMPurify.sanitize(tooltip, {ALLOWED_TAGS: ['strong','br'], ALLOWED_ATTR: ['style']});`

Reply:
Found it!

Reply:
Let me know if you had something else in mind for a fix.

Reply:
Oh, I hadn't seen your message yesterday. I guess the page didn't update while I had it open. But now you approved, so I'll go ahead and merge it.